### PR TITLE
Update UG for v1.4

### LINKED ISSUE #382: remove italics or
check if applicable

### LINKED ISSUE #390: Update header in DG
Remove alternatives considered in the header
Move Aspect to one level below design considerations

### PR DESCRIPTION
**Pull request by shawn-nyk**
Addresses #280 
Addresses #287 
Addresses #294 
Addresses #324

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -272,14 +293,26 @@ Format: `find com KEYWORD [ANOTHER_KEYWORD]...`
...
-     ![FindComGoogleSS](images/FindComGoogleSS.png) <br />
+     ![FindComGoogleSS](images/ug-company/FindComGoogleSS.png) <br />
+     <p align="center">Figure 7: Result of finding companies (example 1)</p>
      
      And executing `find com google facebook` will return the following: <br />
-     ![FindComGoogleFacebookSS](images/FindComGoogleFacebookSS.png) <br />
+     ![FindComGoogleFacebookSS](images/ug-company/FindComGoogleFacebookSS.png) <br />
+     <p align="center">Figure 8: Result of finding companies (example 2)</p>
 
 #### Listing all companies: `list com`
```
Comment (keanecjy) [inline docs/UserGuide.md:307]: I think we are still missing the pictures for delete and list right? Probably should put them in first before numbering the figures?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -191,6 +191,22 @@ This represents a company, any company, but typically one you are interested in
...
+
+<p align="center">Figure 2: Step 1 for all Company commands, with an example</p>
+
+Step 2: Press 'Enter' on your keyboard to execute the command. Upon successful execution, you will see the results of 
+your command reflected immediately in the app. An appropriate result message will also be displayed in the result 
+display.
+
+![GeneralStep2MarkUp](images/ug-company/GeneralStep2MarkUp.png)
+
+<p align="center">Figure 3: Step 2 for all Company commands, with an example</p>
```
Comment (keanecjy) [inline docs/UserGuide.md]: I think the figure number and description should be italicized since it is a side information?

**Review by keanecjy — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (shawn-nyk) [reply docs/UserGuide.md:307]: My bad, I was not certain if we were adding picture for delete and list at first. I've added them in!

File: docs/UserGuide.md (hunk 2 above)
Comment (shawn-nyk) [reply docs/UserGuide.md]: As per today's discussion, will do away with these captions instead. Thanks for pointing this out though!

**Comment by shawn-nyk** [comment]
Updated and now:
Resolves #382 
Resolves #387 
Resolves #390

**Review by keanecjy — APPROVED** [review]

**Pull request merged**